### PR TITLE
Expose host trusted cert store to EEs by default

### PR DESCRIPTION
**Pull request by tchellomello**
##### SUMMARY
Expose host trusted cert store to EEs by default

Related: https://github.com/ansible/awx/issues/10787

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

 - API
 - UI
 - Collection

##### AWX VERSION

```
devel
```

### CODE REVIEW

**Review by CFSNM — APPROVED** [review]

**Review by CFSNM — APPROVED** [review]
Great addition!

**Review by sarabrajsingh — APPROVED** [review]

**Comment by AlanCoding** [comment]
What's going on with this versus https://github.com/ansible/awx/pull/11812?

File: awx/settings/defaults.py [hunk 1]
```diff
@@ -1001,3 +1001,8 @@ def IS_TESTING(argv=None):
 # For example, to disable SELinux in containers for podman
 # DEFAULT_CONTAINER_RUN_OPTIONS = ['--security-opt', 'label=disable']
 DEFAULT_CONTAINER_RUN_OPTIONS = ['--network', 'slirp4netns:enable_ipv6=true']
+
+# List of default exposed container volume mounts
+# This is useful when the EE container needs to access host/node information
+# like /etc/pki/ca-trust/source/anchors
+DEFAULT_CONTAINER_VOLUME_MOUNTS = ['/etc/pki/ca-trust:/etc/pki/ca-trust:O', '/usr/share/pki:/usr/share/pki:O']
```
Comment (AlanCoding) [inline awx/settings/defaults.py:1008]: These folders might not exist on some machines, no?

Comment (shanemcd) [reply awx/settings/defaults.py:1008]: Both of these are on every RHEL and RHEL-ish machine I've ever used. Have you seen any examples where they are missing?

**Comment by shanemcd** [comment]
> What's going on with this versus https://github.com/ansible/awx/pull/11812?

Good call @AlanCoding, I forgot I did that. This PR is also missing a commit that is needed to avoid breaking our development environment. I'm going to close this PR in favor of that one.

**Pull request closed without merging**